modules/ROOT/pages/legacy-search-experience.adoc

```diff
@@ -0,0 +1,311 @@
+|League resource |Endpoint display name
+|Information about a league |Endpoint description
+|POST |Endpoint method
+|create team |Endpoint method description
+|offset |Query parameter name, Query parameter description
+|leagueid |URI parameter name
+|team |Type name
+|city |Type property name
+|leagues leagueId teams |Endpoint path
+|`/leagues/LEAGUE_ID/teams` |Endpoint path
+|===
+
+Note that the last two searches are the same, because special characters are removed before searching.
+
+== Search for Phrases 
+
+You can search for case-insensitive text that begins with each word you specify, put a space between words. This is similar to putting the `*` wildcard regular expression after each word. For example, searching for `mq module` finds any asset that has text that starts with `mq` or `module`.
+
+Searching for a multiple word phrase shows assets that have every word in the phrase.
+
+If you separate search words with any non-alphanumeric character other than a space, Anypoint Exchange converts the character to a space before searching.
+
+
+=== Search Separators
+
+The search experience uses search separators to find Exchange assets that contain one or more terms.
+
+If you input search words connected by non-alphanumeric symbols, Exchange replaces the symbols with spaces before searching for the term. The search string only succeeds if both words are present in the result. This is the same as the `word1 AND word2` relationship.
+
+Example search terms:
+
+[source,text,linenums]
+----
+sales:connect
+Sales-Connect
+connect:/sales
+-connect -sales
+----
+
+This table shows how each search term is interpreted based on the values in the asset name,
+asset ID, and tags for an asset. If the search term maps to the values correctly, a match occurs. If not,
+the reason is provided.
+
+[%header%autowidth.spread]
+|===
+|Asset Name |Asset ID |Tags |Matches?
+|Salesforce API |salesforce-api |mule-Connector |Yes
+|Salesforce Connector |salesforce-connector |REST |Yes
+|Validations API |validations-api |salesforce-connecting |Yes
+|Optimized API |optimized-api |resource:/connect, salesforce:enabled |Yes
+|Cisco Sales API  |cisco-sales-api |Connecting, Productive |Yes
+|Cisco Connector |cisco-connector |com.cisco.connector.sales
+|Yes, because the tag is split into separate terms.
+|Salesforce API |salesforce-api |muleconnector, MuleConnector |No. `connect` was not found.
+|Salesforce Connec |salesforce-connec |REST |No. `connect` was not found.
+|Validations API |validations-api |connector |No. `sales` was not found.
+|Cisco Presales API |cisco-presales-api |Connector, api |No. `sales` was not found.
+|===
+
+=== Phrase Search Separators
+
+Use phrase search to find a group of terms inside quotation marks. Phrase searches match an asset only if all the exact terms inside quotes appear in consecutive order.
+
+If you input any of these searches in Exchange, Exchange replaces non-alphanumeric symbols with spaces before searching for assets.
```

Comment:
I guess "input" as a verb works, but it grates on me :)

Reply:
fixed